resources/asdf-format.org/manifests/transform-1.5.0.yaml

```diff
@@ -675,6 +675,11 @@ tags:
   title: Two dimensional Sersic surface brightness profile.
   description: |-
     Two dimensional Sersic surface brightness profile.
+- tag_uri: tag:stsci.edu:asdf/transform/general_sersic2d-1.0.0
```

Comment:
It appears the 1.5.0 manifest is already in use by asdf-astropy 0.4.x: https://github.com/astropy/asdf-astropy/blob/b3dafcc8e4f400ec667ba538b9f156d4c5279e6f/asdf_astropy/extensions.py#L402
Would you add this to a new `transform-1.6.0.yaml`? That way users with an out-of-date asdf-astropy that attempt to read files that contain this new transform will get a warning about an out-of-date asdf-astropy extension.

Reply:
Feel free to do this, however releasing a whole new transform manifest is a bunch of work that I don't have time to do.